lib/service/tracking/tracking_cubit.dart

```diff
@@ -16,3 +16,38 @@ extension LandingPageTracking on TrackingCubit {
     trackingRepository.track(LandingPageEvent.onPressedStartTrading());
   }
 }
+
+extension TradePageTracking on TrackingCubit {
```

Comment:
There should be one extension / feature layer (Login/Trade/Pool/settings/...). 
Please unify.